Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -1614,18 +1622,18 @@ public void OnSourceLost(SourceStateEventData eventData)
 
                 pointerMediators.Remove(eventData.SourceId);
 
+
                 for (var i = 0; i < eventData.InputSource.Pointers.Length; i++)
                 {
+                    var gazePointer = CoreServices.InputSystem.GazeProvider?.GazePointer;
                     // Special unregistration for Gaze
-                    if (gazeProviderPointingData?.Pointer != null && eventData.InputSource.Pointers[i].PointerId == gazeProviderPointingData.Pointer.PointerId)
+                    if (gazePointer != null && eventData.InputSource.Pointers[i].PointerId == gazePointer.PointerId)
                     {
-                        // If the source lost is the gaze input source, then reset it.
+                        // If the source lost is the gaze input source, clear gazeProviderPointingData.
                         if (eventData.InputSource.SourceId == CoreServices.InputSystem.GazeProvider?.GazeInputSource.SourceId)
                         {
-                            gazeProviderPointingData.ResetFocusedObjects();
```

Comment:
Is this removed because it basically didn't do anything since we `null` out the pointing data in the next line? I guess the only thing it _might_ do is call `OnPreCurrentPointerTargetChange` on the gaze pointer but...that method is empty so ¯\\\_(ツ)_/¯

Reply:
Since the type changed from PointerData gazeProviderPointingData to PointerEventData gazeProviderPointingData, the ResetFocusedObjects() function was no longer applicable.

We're changing the type because using PointerData tied all of our queries to whatever gazeProvider was initialized with MRTK on initialization, rather than the one associated with CoreServices.InputSystem.GazeProvider?.GazePointer, which generally made things more confusing (i.e. we were always cross referencing  CoreServices.InputSystem.GazeProvider for the source of truth)